static/css/pursuit.css

```diff
@@ -619,6 +619,59 @@ ol li {
 .result__actions__item + .result__actions__item {
   margin-left: 0.65em;
 }
+.result__omission {
```

Comment:
https://github.com/purescript/purescript/issues/3016 if you're wondering what's going on